usagov/usagov-2021
# Usagov 364 federal agency index changes

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-364

## Description

Files changed:
config/sync/views.view.federal_agencies.yml 
views-view-summary-federal-agencies.html.twig 
views-view-list-federal-agencies.html.twig 

SubTasks: 
Task 1: The Text (plain, long) field type doesn't allow links. Everything that is saying to create a new field with a Text (formatted, long) field type and migrate the data. Or to use a module (which from what I can tell are just automatically creating that new field and migrating your data in one step);  Instead a) the link was updated to a relative link of “/states.” 
Task  2: views-view-summary-federal-agencies.html.twig and views-view-list-federal-agencies.html.twig have been updated to exclude “A” agencies from query path generation 
Task 3: The < and > "arrows" at the bottom of /agency-index are now linked correctly. Edits made in views-view-list-federal-agencies.html.twig 
Task 4:  Link should read “More information about …” NOT  “More information on …” in views-view-list-federal-agencies.html.twig 


Testing on /agency-index:
1. [Get contact information for each state and territory.](http://localhost/states) should now link to and take you to “/states”
2. Next letter after A should show A on the bottom and both A and < should link and take you to /agency-index NOT /agency-index?letter=a
    1. Last letter should show A on the bottom and both A and > should link and take you to /agency-index NOT /agency-index?letter=a
3. Select any letter to test — < and > should point to the same url as the letter shown points to 
4. Select any letter to test -> Select a directory -> Link should read “More information about …” NOT  “More information on …”


## Checklist for the Developer



## Checklist for the Peer Reviewers
